internal/layers/layers.go

```diff
@@ -302,7 +311,8 @@ func UnmountContainerLayers(ctx context.Context, layerFolders []string, containe
 	return retError
 }
 
-func computeV2Layers(ctx context.Context, vm *uvm.UtilityVM, paths []string) (layers []hcsschema.Layer, err error) {
+// ConvertToV2LayerFormat converts host paths corresponding to container layers into HCS schema V2 layers
+func ConvertToV2LayerFormat(ctx context.Context, vm *uvm.UtilityVM, paths []string) (layers []hcsschema.Layer, err error) {
```

Comment:
Let's just get rid of "V2" from the name here. Maybe just call it `GetHCSLayers`? I don't think we have any corresponding `ConvertToV1LayerFormat`, so no need to keep this confusing naming around.

Reply:
Updated.